Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Scripts/Networking/Forge/Networking/FacepunchP2PServer.cs

```diff
@@ -0,0 +1,731 @@
+﻿#if FACEPUNCH_STEAMWORKS
+using BeardedManStudios.Forge.Networking.Frame;
+using BeardedManStudios.Threading;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using Steamworks;
+
+namespace BeardedManStudios.Forge.Networking
+{
+	public class FacepunchP2PServer : BaseFacepunchP2P, IServer
+	{
+		public List<string> BannedAddresses { get; set; }
+
+		/// <summary>
+		/// Used to determine if this server is currently accepting connections
+		/// </summary>
+		public bool AcceptingConnections { get; private set; }
+
+		public Dictionary<SteamId, SteamNetworkingPlayer> steamPlayers = new Dictionary<SteamId, SteamNetworkingPlayer>();
```

Comment:
A bit of documentation for this please :)

Reply:
Done